### ISSUE TITLE
Make profile icon accessible

### ISSUE THREAD

**Issue by sabakhilji** [saving-satoshi/saving-satoshi#171]
In order to make the project more accessible. I would like to suggest 1 change here:
Naming the profile button in the navbar.
This button is not accessible with the screen reader as it is not named.

Check out this screenshot for code change:
![Screenshot (66)](https://user-images.githubusercontent.com/52318459/220689157-9a80bf0e-736f-4e6a-b1ed-72ded50751cc.png)
 
I would like to suggest adding a name to this UserButton:
   ```
      <button
        onClick={() => setOpenSignInModal(true)}
        className="text-grey-300 ml-4 h-10 w-10 cursor-pointer"
      >
        <Avatar avatar={user.avatar} size={30} />
      </button>
    ) : (
      <button
        onClick={() => toggleModal(true)}
        className="text-grey-300 ml-4 h-10 w-10 cursor-pointer"
      >
        <UserIcon />
      </button>
  ```

Note: I used the narrator (screen reader for windows) to test the accessibility of this page.

**Comment by ecurrencyhodler** [comment]
Hey Saba! Thanks for submitting this issue. @afterburn Curious on your thoughts. Doesn't seem like a big change. What do you think?

**Comment by afterburn** [comment]
Hey @sabakhilji, good suggestion! Feel free to add these via `aria-label` or if you have a better idea, feel free to create a PR. 😁

**Comment by sabakhilji** [comment]
Sure, thanks!
